src/components/EmptyState/index.js

```diff
@@ -49,11 +50,13 @@ export class EmptyState extends React.Component {
                 'Morbi malesuada nulla nec purus convallis consequat. Vivamus id mollis quam. Morbi ac commodo nulla. In condimentum orci id nisl volutpat bibendum. Quisque commodo hendrerit lorem quis egestas. Maecenas quis tortor arcu. Vivamus rutrum nunc non neque consectetur quis placerat neque lobortis.'}
             </p>
           </StackItem>
+          {footer ? 
```

Comment:
This ternary is getting a little hard to read for it's complexity. To make it more readable, I think it would be helpful to shift it up into it's own function between the `contructor` and `render` functions where you can use a full if statement rather than a ternary operator to render this content. That way in place of this conditional rendering here on line 53, you could just have something like `renderFooter()` which would render out either the button or the footer.